packages/ember-glimmer/tests/utils/abstract-test-case.js

```diff
@@ -48,7 +48,7 @@ export function moduleFor(description, TestClass, ...mixins) {
 
   QUnit.module(`[${packageName}] ${descriptionWithoutPackagePrefix}`, {
     setup() {
-      context = new TestClass();
+      context = new TestClass(assert);
```

Comment:
I see. In that case if it's literally just a global, there is no point passing it as a constructor arg. We can just do

```javascript
 export class TestCase {
   constructor() {
      this.assert = QUnit.Assert;
      ...
```